src/expected_loglik.jl

```diff
@@ -73,9 +73,20 @@ function expected_loglik(
     # (see e.g. en.wikipedia.org/wiki/Gauss%E2%80%93Hermite_quadrature)
     xs, ws = gausshermite(gh.n_points)
     # size(fs): (length(y), n_points)
-    fs = √2 * std.(q_f) .* xs' .+ mean.(q_f)
-    lls = loglikelihood.(lik.(fs), y)
-    return sum((1 / √π) * lls * ws)
+    return sum(
+        Broadcast.instantiate(
+            Broadcast.broadcasted(q_f, y) do q, y
+                μ = mean(q)
+                σ = std(q)
+                sum(Broadcast.instantiate(
+                    Broadcast.broadcasted(xs, ws) do x, w
+                        f = sqrt2 * σ * x + μ
+                        loglikelihood(lik(f), y) * w
+                    end,
+                ))
+            end,
+        ),
+    ) * invsqrtπ
```

Comment:
minor nit: just to make it a bit easier to follow, I'd move the `* invsqrtπ` to the front (`return invsqrtπ * sum(...`) - at first I thought you had just dropped it, took me a while to find it :sweat_smile: